### DISCUSSION TITLE
When can the problem for limited internal console buffer size in R be solved?

### DISCUSSION THREAD

**Discussion by LR1933** [posit-dev/positron#4262] [Q&A] [answered]
Positron is great！！！ But it can't run multiple lines of code at once. The console reports` Can't pass console input on to R, it exceeds R's internal console buffer size.`

I know it's still a work in progress. I'd like to know when this limitation will be resolved.

Also, if could, I'd like to know if there is an approximate date for the official release. I'm really looking forward to the official release.

**Comment by jmcphers** [comment, marked as answer]
This is a known issue! You can follow it here to get updates: https://github.com/posit-dev/positron/issues/1326

Sorry, we don't have a public timeline for the stable release yet. Stay tuned, we'll be sharing more when we get closer!
